### ISSUE TITLE
Reading Group

### ISSUE THREAD

**Issue by ghettodev** [smartcontractresearchforum/research_team#117]
- gets people socialized
mentoring etcv

**Comment by ghettodev** [comment]
link to proposal {]

**Comment by zubepaul** [comment]
Proposal document updated.

**Comment by zubepaul** [comment]
Overview document that can be used as an introduction or pinned post has been started by Elizabeth

**Comment by ebethbarnes** [comment]
I've drafted up a [pinned proposal here](https://docs.google.com/document/d/1QQpM80SOCHtfjIjpiJVMWj_w3tpxRclC8vkX9Vk-39o/edit?usp=sharing). Feedback welcome!

**Comment by zubepaul** [comment]
@ebethbarnes I asked one question in the document. I do think this is a good pinned post template as reading groups happen in the chat.

**Comment by zubepaul** [comment]
@ghettodev Is this something that can be moved to "approved"?

**Comment by zubepaul** [comment]
Moved to approved

**Comment by zubepaul** [comment]
Proposal doc updated with Announcement language link. https://docs.google.com/document/d/1e2wI1UVc3Ifjj-PC0wjE-6wgwlaf_6BXJDd8FbwoHZk/edit#

**Comment by zubepaul** [comment]
Update on GitHub and Forum
Discuss in chat 8/2

**Comment by zubepaul** [comment]
Documentation updated, re-review in Ops for repo

**Comment by zubepaul** [comment]
Reading group GitHub pull request made. Engage with chat to discuss the frequency of participation for the first reading group.

**Comment by ghettodev** [comment]
PRs P'd. Added a link to the README. I'm guessing that once the landing page plan is finished we will have a better section to put ongoing programs/initiatives into.

**Comment by zubepaul** [comment]
Based on community call and conversations with @eleventh19 , I updated the outcome of this project. Eventually this should be a model and learning experience of how the community can run reading groups and ultimately produce documentation that would help a person suggest, start, and run a reading group on their own through the forum. This might necessitate the creation of a reading group or community category on the forum.

**Comment by zubepaul** [comment]
I will be updating the process in preparation for iteration #2. Call for research summary is next step from iteration #1.

**Comment by eleventh19** [comment]
Related activity, @Ashley-Y-Lin and @twan-dev are working on some revised docs for the club engagement which is starting off with a reading group idea. #169. most recent [project doc](https://docs.google.com/document/d/13ATuQut-y3x7-MNh23c5jjHfEFgyAzO2OoYuYLm33bY/edit) from that project and the current [brainstorming doc](https://docs.google.com/document/d/19d57PRgwghcA1HBvdyJbRY5dZOAHvLi-8fH_QAkpZRs/edit#heading=h.8h03eynrkp61) for ideas of what the trials can look like this semester.

**Comment by zubepaul** [comment]
@eleventh19 We should discuss the role that Lorretta can make this work.

**Comment by eleventh19** [comment]
that didn't end up happening. we're shooting to go over [this paper](https://fermatslibrary.com/s/bitcoin) on 12/9 in the community call. @LorrettaGS need to announce that to the community this week

**Comment by LorrettaGS** [comment]
@eleventh19 I assumed that this was the paper for Blockchain Basics? or can we now combine the Reading Group and Blockchain Basics. 
Which I now think is a great idea...

**Comment by eleventh19** [comment]
@zubepaul should we do a breakout on community at some point in the coming weeks to start planning overall events / plan going forward? or put on hold for now?

**Comment by eleventh19** [comment]
We are going to do a ['How to SCRF Better' reading group](https://docs.google.com/document/d/1BjsOH_9e1_gzFLoWCJrwnfAOq_Y5zFlREdhZNu7zEuE/edit#) to start. This is slightly different than the initial idea, so this card will remain open until there is a plan in place to operationalize a wider reading group.

**Comment by eleventh19** [comment]
Message Umar posted in the community channel. Besides that, we're getting ready to post in the forum to get a new book chosen 

Hi everyone! Due to popular demand, we're back with more sessions for the Impact Networks Reading Group!📚  🎉

Our first discussion session (on June 27th) focused on Part 1 of the book and was at an inconvenient time for APAC residents. We’re having two more reading group sessions, at friendlier times, with the goal of continuing our earlier discussion and finishing the book.

Thank you to @isss111  @eugene  @paulzed  for putting these sessions together!

Upcoming Reading Group Sessions
Date: August 11th at 9am EST for 60 minutes
Goal: Discuss Part 1 (Chapters 1 to 4) of Impact Networks in the context of SCRF and our mission

Date: August 24th at 11am EST for 60 minutes
Goal: Discuss Part 2 (Chapters 5 to 10) of Impact Networks in the context of SCRF and our mission

Register Here for a Calendar Invite: https://forms.gle/xhNkFAPXU9j2qSUp6
More Info on the Reading Group and Book: https://www.smartcontractresearch.org/t/reading-group-impact-networks-june-27th-community-call-june-30th/1628/
